packages/generator/test/renderers/renderers.test.ts

```diff
@@ -88,51 +88,4 @@ describe("Renderers", () => {
       );
     }).timeout(10000);
   });
-
```

Comment:
`js-yaml` is no longer needed. It should be removed from this file and removed as a dependency.